api/mailinabox.yml

```diff
@@ -0,0 +1,1714 @@
+openapi: 3.0.3
+info:
+  title: Mail-in-a-Box
+  description: 'Mail-in-a-Box API HTTP specification.'
+  contact:
+    name: Richard Willis
+    url: https://github.com/badsyntax
+    email: willis.rh@gmail.com
+  license:
+    name: MIT
+    url: https://badsyntax.mit-license.org/
```

Comment:
Update license to match https://github.com/mail-in-a-box/mailinabox/blob/master/LICENSE